esp-hal-common/src/rtc_cntl/mod.rs

```diff
@@ -754,20 +759,21 @@ pub fn get_reset_reason(cpu: Cpu) -> Option<SocResetReason> {
 
 pub fn get_wakeup_cause() -> SleepSource {
     // FIXME: check s_light_sleep_wakeup
-    // https://github.com/espressif/esp-idf/blob/afbdb0f3ef195ab51690a64e22bfb8a5cd487914/components/esp_hw_support/sleep_modes.c#L1394
-    if get_reset_reason(Cpu::ProCpu).unwrap() != SocResetReason::CoreDeepSleep {
-        return SleepSource::Undefined;
-    }
+    // https://github.com/espressif/esp-idf/blob/afbdb0f/components/esp_hw_support/sleep_modes.c#L1394
+    // FIXME: un-comment block when H2's `SocResetReason` enum is populated
+    // if get_reset_reason(Cpu::ProCpu).unwrap() != SocResetReason::CoreDeepSleep {
+    //     return SleepSource::Undefined;
+    // }
```

Comment:
If possible lets populate this enum please; if it's a hassle we can wait, but I'd rather just get it done.

Reply:
Enum was populated and function is uncommented